infra/README.md

```diff
@@ -202,6 +211,56 @@ tags = merge(module.common.default_tags, {
 
 })
 ```
+
+## Destroying Infrastructure
+
+To destroy everything you'll need to undo everythin in reverse.
+
+1. First destroy all your environments by running `terraform destroy` in each of your environment module folders
+
+    ```bash
+    # within each env folder in /infra/envs/ (dev, stage, etc)
+    terraform destroy
+    ```
+
+2. Then to destroy the backends, first you'll need to add `force_destroy = true` to the S3 buckets, and update the
+  lifecycle block to set `prevent_destroy = false`. Then run `terraform apply`
+
+    ```terraform
+    resource "aws_s3_bucket" "tf_state" {
```

Comment:
should we add a comment to the top of these code blocks indicate what file this is in? it's a little difficult to follow.